app/components/QRMob.tsx

```diff
@@ -12,34 +12,32 @@ interface QRMobProps {
   value: string;
 }
 
-const useStyles = makeStyles(() => {
-  return {
-    container: {
-      height: '296px',
-      position: 'relative',
-      width: '296px',
-    },
-    icon: {
-      alignItems: 'center',
-      display: 'flex',
-      height: '100%',
-      justifyContent: 'center',
-      left: 0,
-      position: 'absolute',
-      top: 0,
-      width: '100%',
-      zIndex: 1000,
-    },
-    qr: {
-      height: '100%',
-      left: 6,
-      position: 'absolute',
-      top: 6,
-      width: '100%',
-    },
-    root: {},
-  };
-});
+const useStyles = makeStyles(() => ({
+  container: {
+    height: '296px',
+    position: 'relative',
+    width: '296px',
```

Comment:
This is general note unrelated to this PR, but we should loop back at some point and clean up all of these hardcoded widths and heights.